doc/sphinx/source/quickstart/running.rst

```diff
@@ -4,138 +4,88 @@
 Running
 *******
 
-ESMValTool is mostly used as a command line tool. Whenever your
-conda environment for ESMValTool is active, you can just run the command
-``esmvaltool``. See
-:ref:`running esmvaltool <esmvalcore:running>`
-in the ESMValCore documentation for a short introduction.
+ESMValTool is mostly used as a command line tool.
+Whenever your Conda environment for ESMValTool is active, you can run the
+command ``esmvaltool``.
+See :ref:`running esmvaltool <esmvalcore:running>` in the ESMValCore
+documentation for an introduction to the ``esmvaltool`` command.
 
-Running a recipe
-================
+Running your first recipe
+=========================
+
+There is a step-by-step tutorial available in the
+`ESMValTool tutorial <https://esmvalgroup.github.io/ESMValTool_Tutorial/>`_
+on how to run your first recipe. It can be found
+`here <https://esmvalgroup.github.io/ESMValTool_Tutorial/04-recipe/index.html>`_.
 
 An
 `example recipe <https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/examples/recipe_python.yml>`_
 is available in the ESMValTool installation folder as
 ``examples/recipe_python.yml``.
 
-This recipe finds data from CanESM2 and MPI-ESM-LR for 2000 - 2002,
-extracts a single level (850 hPa), regrids it to a 1x1 degree mesh and runs
-a diagnostic script that creates some plots of Air temperature and
-precipitation flux. You can download the recipe from
-`github <https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/examples/recipe_python.yml>`_
-and save it in your project directory as (e.g.) ``recipe_python.yml``
-and then run ESMValTool with
+To run this recipe and automatically download the required climate data
+from ESGF to the local directory ``~/climate_data``, run
 
 .. code:: bash
 
-	esmvaltool run recipe_python.yml --synda-download
+	esmvaltool run examples/recipe_python.yml --offline=False
 
-The ``--synda-download`` option tells ESMValTool to use Synda to search for and download
+The ``--offline=False`` option tells ESMValTool to search for and download
 the necessary datasets.
```

Comment:
- Can we link to this[ user-configuration-file](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/configure.html#user-configuration-file) page that shows how to set these settings in `config-user.yml`?
- Also, as the option `--offline=False` downloads missing data, can we add an example of using two rootpath and two drs (I am not sure if a list of drs is possible) e.g. : 
```yaml
rootpath:
  default: [~/climate_data, /mnt/lustre02/work/ik1017/CMIP6/data/CMIP6]

drs:
  CMIP6: [ESGF, DKRZ]
```
- it is useful to elaborate on `download_dir` and explain why `download_dir` cannot be a the same as CMIP6 directory on a HPC for example `/mnt/lustre02/work/ik1017/CMIP6/data/CMIP6`.

Reply:
I'll try to improve this a bit here, but this section is really intended as just an example of running a first recipe. The full run and configuration options are explained in the linked esmvalcore documentation.

Reply:
>  can we add an example of using two rootpath and two drs (I am not sure if a list of drs is possible)

I think this is not possible. The only way to find downloaded data if you already have your rootpath and DRS set to something else, is by running the tool in `--offline=False` mode.

Reply:
> The only way to find downloaded data if you already have your rootpath and DRS set to something else, is by running the tool in `--offline=False mode`. 

Thank you. So does it mean that I can run a recipe with this config? then esmvaltool uses data in `roothpath` and downloads the missing data too? 
```yaml
# Directory for storing downloaded climate data
download_dir: ~/climate_data

# Disable the automatic download of missing CMIP3, CMIP5, CMIP6, CORDEX,
# and obs4MIPs data from ESGF [true]/false.
offline: false

# Rootpaths to the data from different projects (lists are also possible)
rootpath:
  CMIP6: /mnt/lustre02/work/ik1017/CMIP6/data/CMIP6

# Directory structure for input data: [default]/ESGF/BADC/DKRZ/ETHZ/etc
# See config-developer.yml for definitions.
# comment out/replace as per needed
drs:
  CMIP6: DKRZ
```
Can we add this example somewhere? This might be clear for developers, but I think it will help users a lot.

Reply:
@SarahAlidoost good call!

Reply:
I frequently use two rootpaths, but they use the same dr - How does the data download structure things (have not tested this yet myself)? If it's in just a plain folder it might be useful to add a util tool to recreate commonly used dr structures in your download folder.

Reply:
@bettina-gier the data download drs is ESGF. 

@bouweandela and @valeriupredoi I ran an experiment (see example below) that works. However, as you see I didn't set `ESGF` in `drs` part of the config-user.yml. I am not sure if it is a bug or a feature! 

**Example: when some of the data are missing**
<details>
The [recipe_python.yml](https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/examples/recipe_python.yml) needs these data:

1. cmip5: areacella_fx_CanESM2_historical_r0i0p0.nc
2. cmip5: tas_Amon_CanESM2_historical_r1i1p1_185001-200512.nc 
3. cmip6: tas_Amon_BCC-ESM1_historical_r1i1p1f1_gn_185001-201412.nc

Imagine that we are working on a system where `drs` is according to DKRZ and only data No. 2 is available.
We can set the configuration as:

```yml
# Directory for storing downloaded climate data
download_dir: ~/climate_data

# Disable the automatic download of missing CMIP3, CMIP5, CMIP6, CORDEX,
# and obs4MIPs data from ESGF [true]/false.
offline: false

# Rootpaths to the data from different projects (lists are also possible)
rootpath:
  default: ~/climate_data
  CMIP5: /path_to_CMIP5_DKRZ

# Directory structure for input data: [default]/ESGF/BADC/DKRZ/ETHZ/etc
# See config-developer.yml for definitions.
# comment out/replace as per needed
drs:
  CMIP5: DKRZ
```  
</details>

Reply:
@SarahAlidoost Is this the expected outcome? Or did you expect it to only download CMIP6 because you already set the rootpath and DRS for CMIP5?

Regarding the settings:
- to use automatically downloaded data set `offline: false` and `download_dir: /some/path`
- to use data that is already present on the system use the `rootpath` and `drs` settings

It is not necessary to add the `default: ~/climate_data` project under `rootpath`, though of course you can do this. Note however that this will only result in finding the downloaded data with `offline: true` if you also set the project DRS correctly, e.g. `drs: {CMIP6: ESGF, CMIP5: ESGF}`

Reply:
> @SarahAlidoost Is this the expected outcome? Or did you expect it to only download CMIP6 because you already set the rootpath and DRS for CMIP5?

Sorry for my late reply. It is the expected outcome. Two files (one from cmip5 and another from cmip6) were missing in my example. After running the recipe, only those are downloaded.

> Regarding the settings:
> 
> * to use automatically downloaded data set `offline: false` and `download_dir: /some/path`
> * to use data that is already present on the system use the `rootpath` and `drs` settings
> 

Yes, this is correct. This means that if **some part** of the data is missing, both settings should be available in the recipe (like my example above). I think it would be great if we consider this case as an example and include it in the documentation. 

> It is not necessary to add the `default: ~/climate_data` project under `rootpath`, though of course you can do this. Note however that this will only result in finding the downloaded data with `offline: true` if you also set the project DRS correctly, e.g. `drs: {CMIP6: ESGF, CMIP5: ESGF}`